src/server.js

```diff
@@ -44,6 +45,8 @@ const server = http.createServer(app);
 // serve the static HTML
 app.use(express.static(STATIC_DIR));
 
+app.use('/wallet/*', express.static(WALLET_DIR));
```

Comment:
is this still necessary?

Reply:
It's not used in production currently, no. If someone else ends up hosting or we switch to cloudflare it may be worth keeping, just a decision to make.